src/shared/model/Newsletter.ts

```diff
@@ -11,15 +11,19 @@ export enum Newsletters {
 	DOWN_TO_EARTH = '4147',
 	FIRST_EDITION_UK = '4156',
 	THE_LONG_READ = '4165',
+	WELL_ACTUALLY = '6039',
 	// US newsletters
 	HEADLINES_US = '4152',
 	FIRST_THING_US = '4300',
 	SOCCER_US = '6030',
+	THE_STAKES_US = '6042',
+	TRUMP_ON_TRIAL_US = '6032',
 	// AUS newsletters
 	MORNING_MAIL_AU = '4148',
 	AFTERNOON_UPDATE_AU = '6023',
 	FIVE_GREAT_READS_AU = '6019',
 	THE_CRUNCH_AU = '6034',
+	SAVED_FOR_LATER_AU = '6003',
 	// EU newsletters
 	TECHSCAPE = '6013',
 	THIS_IS_EUROPE = '4234',
```

Comment:
can we remove any of these newsletters if they're no longer used anywhere?

i.e not referenced/not in the newsletters map